nextflow.config

```diff
@@ -52,6 +52,25 @@ profiles {
         conda.cacheDir = "${projectDir}/cache/conda"
     }
 
+    mamba {
+        useMamba = true
+
+        process {
+            conda = "${projectDir}/task-environment.yml"
+
+            beforeScript = """
+            export SAMTOOLS_BINARY=samtools
+            export PICARD_BINARY=picard
+            export JAVA_BINARY=java
+            export MBUFFER_BINARY=mbuffer
```

Comment:
after having experienced issues with a certain mbuffer version, would it make sense to stick to a certain version here as well or is this already taken care of somewhere else?

Reply:
The mbuffer version is fixed in the Conda [task environment](https://github.com/DKFZ-ODCF/nf-bam2fastq/blob/6039bc2858e2168e25b1062b604b9741247a728c/task-environment.yml#L9), which is also used by the containers.